index.html

```diff
@@ -885,6 +856,158 @@ <h5>Timing Properties</h5>
+  
+    &lt;descriptor-token&gt;
+    : <a href="https://www.w3.org/TR/xmlschema-2/#NMTOKEN">xsd:NMtoken</a>
+
+    &lt;descriptor-delimiter&gt;
+    : ;  # SEMI-COLON U+003B
+              </pre>
+            </div>
+            <p><code>&lt;<a>content-descriptor</a>&gt;</code> has values that are delimiter separated ordered lists
+              of tokens.</p>
+            <p>A <code>&lt;<a>content-descriptor</a>&gt;</code> value <em>A</em> is a <dfn data-abbr="sub-type">content descriptor sub-type</dfn>
+              of another <code>&lt;content-descriptor&gt;</code> value <em>B</em> if B's ordered list of tokens is
+              present at the beginning of A's ordered list of tokens.
+            </p>
+            <aside class="example">
+              <table class="data">
+                <caption>Table demonstrating example values of <code>&lt;<a>content-descriptor</a>&gt;</code>
+                  and whether each is a  <a>sub-type</a> of the other.
+                </caption>
+                <thead>
+                  <tr>
+                    <th><code>&lt;content-descriptor&gt;</code> A</th>
+                    <th><code>&lt;content-descriptor&gt;</code> B</th>
+                    <th>Is B a <a>sub-type</a> of A?</th>
+                  </tr>
+                </thead>
+                <tbody>
+                  <tr>
+                    <td><code>visual;text</code></td>
+                    <td><code>visual</code></td>
+                    <td>No</td>
+                  </tr>
+                  <tr>
+                    <td><code>visual;text</code></td>
+                    <td><code>visual;text</code></td>
+                    <td>Yes</td>
+                  </tr>
+                  <tr>
+                    <td><code>visual;text</code></td>
+                    <td><code>visual;text;location</code></td>
+                    <td>yes</td>
+                  </tr>
+                </tbody>
+              </table>
+              <p>For example, in this table A could be one of the values listed in <a>Script Represents</a> property,
+                and B could be the value of a <a>Represents</a> property.</p>
+            </aside>
+            <p>The permitted values for <code>&lt;<a>content-descriptor</a>&gt;</code>
+              are either those listed in the following <a>registry table</a>, or can be user-defined:</p>
+              <div class="registry-table-section">
+                <table class="data">
+                  <caption><a>Registry table</a> for the <code>&lt;<a>content-descriptor</a>&gt;</code> component
+                    whose Registry Definition is at <a href="#registry-table-content-descriptor"></a></caption>
+                  <thead>
+                    <th><code>&lt;<a>content-descriptor</a>&gt;</code></th>
+                    <th>Status</th>
+                    <th>Description</th>
+                    <th>Example usage</th>
+                    <th>Notes</th>
```

Comment:
The notes column seems superfluous given the description and example usage.

Reply:
I'd prefer to retain it.